src/encoded/static/components/search.js

```diff
@@ -378,8 +378,8 @@ const DatasetComponent = (props, reactContext) => {
 
     // Get the biosample info for Series types if any. Can be string or array. If array, only use iff 1 term name exists
     if (seriesDataset) {
-        biosampleTerm = (result.biosample_term_name && typeof result.biosample_term_name === 'object' && result.biosample_term_name.length === 1) ? result.biosample_term_name[0] :
-            ((result.biosample_term_name && typeof result.biosample_term_name === 'string') ? result.biosample_term_name : '');
+        biosampleTerm = (result.biosample_ontology.term_name && typeof result.biosample_ontology.term_name === 'object' && result.biosample_ontology.term_name.length === 1) ? result.biosample_ontology.term_name[0] :
+            ((result.biosample_ontology.term_name && typeof result.biosample_ontology.term_name === 'string') ? result.biosample_ontology.term_name : '');
```

Comment:
I think these tests will always fail. You can likely replace all of this with:

`biosampleTerm = result.biosample_ontology.term_name;`

…or just use `result.biosample_ontology.term_name` instead of biosampleTerm where needed.

Reply:
This is OK to keep. DatasetComponent is never called anymore, but could conceivably be in very unusual circumstances. So you could argue that result.biosample_ontology should be checked for existence before executing this line, but this would nearly never come up.